src/reducers/Entities/SensorReducer.js

```diff
@@ -72,6 +73,18 @@ export const SensorReducer = (state = initialState(), action) => {
       return initialState();
     }
 
+    case SENSOR_ACTIONS.REPLACE: {
+      const { payload } = action;
+      const { macAddress, id } = payload;
+      const { byId, editingId } = state;
+
+      const oldSensor = byId[editingId];
+      const newSensor = { ...oldSensor, macAddress, id };
+      const newById = { ...byId, [id]: newSensor };
+
+      return { ...state, byId: newById, replacedId: editingId, editingId: id };
+    }
```

Comment:
this was actually a kinda cool thing to just switch around the ids and it seemed to work (HOPEFULLY 🤞 )